adapters/smaato/banner.go

```diff
@@ -0,0 +1,21 @@
+package smaato
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+)
+
+func extractAdmBanner(adMarkup string, curls []string) string {
+	var clickEvent string
+	if len(curls) > 0 {
+		var clicks strings.Builder
+		for _, clicktracker := range curls {
```

Comment:
How many curls do you expect the adm to contain?

Reply:
unlikely more than 3
I believe atm we mostly supply 1

Reply:
I also changed it to only execute this logic when `curls` field is sent

cc @el-chuck